app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
:smile: I never wanted the `.directive` postfix to be removed. Yeah, we wrote almost a bible worth of text out of a small thing, lol.

Summary:
I wanted you to drop the `uc` prefix from everywhere. I was wrong. You are right that it should be in the directive name. Everywhere else it should be removed. That's how it is in the angular.io style guide, the johnpapa style guide, and that's how all the directives and their respective files and modules are named in city_info and upcoming.

So in short:
Rename the directory `ucBackground` to `background`
Rename the file `ucBackground.module.js` to `background.module.js`
Rename the file `ucBackground.directive.js` to `background.directive.js`
Rename the module name in [this file](https://github.com/heinoapt/upcoming/blob/7dacb82248af452cafda6d23b6d5891f391560c4/app/js/shared/ucBackground/ucBackground.module.js#L5) from `'shared.ucBackground'` to `'shared.background'`